### PR TITLE
DEV-352: Add --version to the Oven

### PR DESCRIPTION
**Pull request by Atlante45**
[JIRA Ticket](https://highfidelity.atlassian.net/browse/DEV-352)

Add --version option to the oven binary

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20824/HighFidelity-Beta-PR16051-97038a7.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20824/HighFidelity-Beta-Interface-PR16051-97038a7.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20824/HighFidelity-Beta-PR16051-97038a7.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20824/HighFidelity-Quest-Beta-PR16051-97038a7.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20824/HighFidelity-Beta-PR16051-97038a7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20824/HighFidelity-Beta-Interface-PR16051-97038a7.exe

File: tools/oven/src/OvenCLIApplication.h [hunk 1]
```diff
@@ -21,7 +21,14 @@ class OvenCLIApplication : public QCoreApplication, public Oven {
 public:
     OvenCLIApplication(int argc, char* argv[]);
 
+    static void parseCommandLine(int argc, char* argv[]);
```
Comment (huffman) [inline tools/oven/src/OvenCLIApplication.h]: Is this static to avoid instantiating the full OvenCLIApplication? Or is there another reason?

Comment (Atlante45) [reply tools/oven/src/OvenCLIApplication.h]: Yeah, that's it.
Otherwise, it prints tons of stuff setting up and taking down.

Comment (Atlante45) [reply tools/oven/src/OvenCLIApplication.h]: I just did the same thing I did for the DS and AC a while back.

**Pull request merged**